Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -509,17 +509,18 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
       .MarkAsAdvanced();
 
   obj.AddCondition("AngleOfDisplacement",
-                   _("Angle of movement"),
-                   _("Compare the angle of displacement of an object"),
-                   _("Angle of displacement of _PARAM0_ is _PARAM1_ (tolerance "
+                   _("Angle of movement (using forces)"),
+                   _("Compare the angle of movement of an object using forces."
+                     " according to the forces."),
```

Comment:
Typo here :) "Compare the angle of movement of an object according to the forces applied on it."?